library/python/e2e_test.py

```diff
@@ -0,0 +1,136 @@
+import threading
```

Comment:
this depends on `gevent` and `python3`. need to figure out how to do reproducible python builds in Bazel